Appraisals

```diff
@@ -1,15 +1,11 @@
-appraise 'rails-6.1' do
-  gem 'rails', '~> 6.1.0'
-end
-
-appraise 'rails-7.0' do
-  gem 'rails', '~> 7.0.0'
-end
-
 appraise 'rails-7.1' do
   gem 'rails', '~> 7.1.0'
 end
 
 appraise 'rails-7.2' do
+  gem 'rails', github: 'rails/rails', branch: '7-2-stable'
+end
+
+appraise 'rails-8.0' do
```

Comment:
This is more `rails main/edge` rather than 8.0, because it can be `8.1`, `8.2` `9.0` etc. It represents the  "future of rails".